### DISCUSSION TITLE
CC villager building

### DISCUSSION THREAD

**Discussion by TRMTZD** [cc-tweaked/CC-Tweaked#1754] [Q&A] [answered]
How do I disable CC villager buildings?

**Comment by SquidDev** [comment, marked as answer]
I believe this is a feature of [AdvancedPeripherals](https://github.com/IntelligenceModding/AdvancedPeripherals) (rather than CC:T itself), so @SirEndii is probably better placed to answer this.

There should be a `enableVillagerStructures` option inside your server config file (AFAIK, located at `<world folder>/serverconfigs/Advancedperipherals/world.toml`).

**Reply by TRMTZD** [reply]
Ohhh okay... I wasn't sure. Thanks for the quick reply I really appreciate it! Thanks!!

**Reply by SirEndii** [reply]
Yeah that's right
You can adjust the rarity of the buildings there
